symengine/mp_class.h

```diff
@@ -711,6 +745,11 @@ inline double mp_get_d(const integer_class &i)
     return i.convert_to<double>();
 }
 
+inline void mp_set_d(integer_class &i, double a)
+{
+    boost::multiprecision::mpz_set_d(i, a);
```

Comment:
That returns `/home/travis/build/ShikharJ/symengine/symengine/mp_class.h:750:9: error: no match for ‘operator=’ in ‘i = a’`